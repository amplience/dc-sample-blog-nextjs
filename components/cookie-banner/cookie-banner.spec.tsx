/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import CookieBanner from './cookie-banner';

describe('CookieBanner', () => {
  test('renders cookie consent banner', async () => {
    const component = await renderer.create(<CookieBanner />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
