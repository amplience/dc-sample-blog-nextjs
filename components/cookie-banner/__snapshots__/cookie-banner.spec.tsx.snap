// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CookieBanner renders cookie consent banner 1`] = `
<div
  className="CookieConsent"
  style={
    Object {
      "alignItems": "center",
      "background": "rgba(255, 255, 255, 0.9)",
      "bottom": "0",
      "boxShadow": "20px 20px 20px 10px #000",
      "color": "#333",
      "flexDirection": "column",
      "position": "fixed",
      "textAlign": "center",
      "width": "100%",
      "zIndex": 1,
    }
  }
>
  <div
    className="cookie-banner-content"
    style={
      Object {
        "padding": "20px 0px 10px",
      }
    }
  >
    We use cookies to give you the best possible experience on our website. By continuing to browse this site, you give consent for cookies to be used. For more details please read our
     
    <a
      href="https://amplience.com/privacy-cookie-policy/"
      rel="noreferrer"
      target="_blank"
    >
      CookieÂ Policy
    </a>
    .
  </div>
  <div
    className=""
  >
    <button
      className="cookie-banner-btn"
      id="rcc-confirm-button"
      onClick={[Function]}
      style={
        Object {
          "background": "#fff",
          "border": "3px solid #CCC",
          "borderRadius": "5px",
          "color": "#333",
          "marginBottom": "10px",
        }
      }
    >
      Accept Cookies
    </button>
  </div>
</div>
`;
